app/components/synchronizers/table-item.tsx

```diff
@@ -37,8 +38,42 @@ export function TableItem({
 }: {
   item: Synchronizer;
 }) {
+  // load hooks
+  const submit = useSubmit();
+  const { pathname, search } = useLocation();
+
   const networkAvatar = GetNetworkAvatar(network);
 
+  // define handlers
+  function onClickHandler(action: string) {
+    // prepare data to send in the form
+    const formData = new FormData();
+    formData.append("redirectURL", `${pathname}${search}`);
+    formData.append("eventName", name);
+    formData.append("address", address);
+
+    // send form to delete event action
+    submit(formData, {
+      method: "post",
+      action: `/admin/synchronizers/actions/${action}`,
+    });
+  }
+
+  let action = null;
+  if (status === "running") {
```

Comment:
maybe we should evaluate to use constants for this like `statusRunnin; statusStopped; etc". Just an idea since this kind of statuses are very common among our services